FNML/overview.md

```diff
@@ -6,14 +6,16 @@ we combine <a>RML</a> with declarative function descriptions in <a>FnO</a>.
 <a>FnO</a> consists of -- among others -- <a>function descriptions</a> and <a>execution</a> descriptions.
 We describe these <a>executions</a> -- which link to specific <a>function descriptions</a> -- within the <a>RML mapping</a>.
 
-A [=triples map=] generates triples from input data.
-An [=RML mapping=] uses [=output triples map=]s to generate output triples from input data.
-We use an intermediate [=function triples map=] to generate [=execution triples=] from input data,
-and then use a [=function term map=] to link the output of those execution triples to the actual [=output triples map=],
-so that input data is transformed via a [=function triples map=] and then integrated in the output using a [=function term map=].
+A [=triples map=] "specifies the rules for translating each record" in input data into the output triples. We define [=function triples map=], a specific type of triplesMap (following the same semantics) to generate output of the function from input data. [=function triples map=] includes:
+- Exactly one rml:logicalSource
+- Exactly one fnml:functionTermMap to define the output of the function which is the output of the triplesMap.
+- Exactly one rml:predicateObjectMap to define the execution of the function
+- At least one rml:predicateObjectMap to define the input of the function
```

Comment:
how are you going to distinguish which one defines the execution and which one defines the input of the function?